src/OrchardCore.Modules/OrchardCore.Widgets/wwwroot/Scripts/widgetslist.edit.js

```diff
@@ -1,19 +1,21 @@
-var createEditorUrl = $('#buildEditorUrl').attr("value");
-var widgetTemplate = function (data, prefixesName, prefix, contentTypesName, contentType, zoneName, zone) {
-    return '<div class="widget-template col-md-12">' + data + '<input type="hidden" name="' + prefixesName + '" value="' + prefix + '" /><input type="hidden" name="' + contentTypesName + '" value="' + contentType + '" /><input type="hidden" name="' + zoneName + '" value="' + zone + '" class="source-zone" /></div>';
-};
-function guid() {
-    function s4() {
-        return Math.floor((1 + Math.random()) * 0x10000)
-          .toString(16)
-          .substring(1);
-    }
-    return s4() + s4() + s4() + s4() + s4() + s4() + s4() + s4();
-}
-
 $(function () {
+    //function scoped variables
+    var widgetTemplate = function (data, prefixesName, prefix, contentTypesName, contentType, zoneName, zone) {
+        return '<div class="widget-template col-md-12">' + data + '<input type="hidden" name="' + prefixesName + '" value="' + prefix + '" /><input type="hidden" name="' + contentTypesName + '" value="' + contentType + '" /><input type="hidden" name="' + zoneName + '" value="' + zone + '" class="source-zone" /></div>';
+    };
+    function guid() {
+        function s4() {
+            return Math.floor((1 + Math.random()) * 0x10000)
+                .toString(16)
+                .substring(1);
+        }
+        return s4() + s4() + s4() + s4() + s4() + s4() + s4() + s4();
```

Comment:
Same as my earlier comment

Reply:
I have no intention to change the logic of existing function, just moved function variable inside $() scope to avoid conflict